### PR TITLE
Minor refactor: Be consistent about alias_eliminate_graph! return type

### PR DESCRIPTION
**Pull request by Keno**
Generally, it's bad style to change the return type in a corner cases,
because a lot of the time (as I happed to), users will mostly test
the common case and then when the corner case (in this case no linear
equations in the system) comes up, things break. Simplify things
by being consistent about the return types.

### CODE REVIEW

File: src/systems/alias_elimination.jl [hunk 1]
```diff
@@ -4,7 +4,7 @@ const KEEP = typemin(Int)
 
 function alias_eliminate_graph!(state::TransformationState)
     mm = linear_subsys_adjmat(state)
-    size(mm, 1) == 0 && return nothing, mm, BitSet() # No linear subsystems
+    size(mm, 1) == 0 && return AliasGraph(ndsts(state.structure.graph)), mm, BitSet() # No linear subsystems
```
Comment (YingboMa) [inline src/systems/alias_elimination.jl:7]: This creates unnecessary allocation though

Comment (Keno) [reply src/systems/alias_elimination.jl:7]: I don't think it's worth worrying about.
